### ISSUE TITLE
Don't pass through datafiles on UNCCD reporting tool if affected areas only selected

### ISSUE THREAD

**Issue by azvoleff** [ConservationInternational/trends.earth#596]

**Comment by azvoleff** [comment]
@boc84 - this should now be fully functional in the develop version of Trends.Earth

**Comment by boc84** [comment]
> @boc84 - this should now be fully functional in the develop version of Trends.Earth

This would be version 1.99.7?

**Comment by azvoleff** [comment]
The current development version is 1.99.9 - but always good to update from the [latest zipfile](https://github.com/ConservationInternational/trends.earth#development-version-from-zipfile) or code itself.

When running a development version of the plugin, the version number itself isn't a reliable indicator of having the latest version of the code, as the plugin can change on the develop branch without incrementing the version numbers. For released versions of the plugin (those ending in an even number), that isn't the case, but for develop there are too many different changes within a single version to be continually incrementing the version number. The best indicator of what version of the plugin you have is the SHA hash on the about screen:

![image](https://user-images.githubusercontent.com/107753/161624522-a33e7db6-f5ed-4691-80f5-83bbb34489c4.png)

Though note that I just realized that there was a bug so the about screen couldn't be accessed... that is now fixed and the best way to check the version in the future.
